packages/vaex-core/setup.py

```diff
@@ -75,14 +60,12 @@ def __str__(self):
     extra_compile_args.append("-mmacosx-version-min=10.9")
 
 # on windows (Conda-forge builds), the dirname is an absolute path
-extension_vaexfast = Extension("vaex.vaexfast", [os.path.relpath(os.path.join(dirname, "src/vaexfast.cpp"))],
+extension_vaexfast = Pybind11Extension("vaex.vaexfast", [os.path.relpath(os.path.join(dirname, "src/vaexfast.cpp"))],
```

Comment:
Not using Pybind11 here:
```suggestion
extension_vaexfast = Extension("vaex.vaexfast", [os.path.relpath(os.path.join(dirname, "src/vaexfast.cpp"))],
```